cmd/openshift-install/gather.go

```diff
@@ -270,22 +270,34 @@ func logClusterOperatorConditions(ctx context.Context, config *rest.Config) erro
 		return errors.Wrap(err, "listing ClusterOperator objects")
 	}
 
+	errorCount:=0
 	for _, operator := range operators.Items {
 		for _, condition := range operator.Status.Conditions {
 			if condition.Type == configv1.OperatorUpgradeable {
 				continue
 			} else if condition.Type == configv1.OperatorAvailable && condition.Status == configv1.ConditionTrue {
 				continue
-			} else if (condition.Type == configv1.OperatorDegraded || condition.Type == configv1.OperatorProgressing) && condition.Status == configv1.ConditionFalse {
+			} else if condition.Type == configv1.OperatorDegraded && condition.Status == configv1.ConditionFalse {
+				continue
+			} else if condition.Type == configv1.OperatorProgressing && condition.Status == configv1.ConditionFalse {
 				continue
 			}
 			if condition.Type == configv1.OperatorDegraded {
 				logrus.Errorf("Cluster operator %s %s is %s with %s: %s", operator.ObjectMeta.Name, condition.Type, condition.Status, condition.Reason, condition.Message)
+				errorCount++
 			} else {
 				logrus.Infof("Cluster operator %s %s is %s with %s: %s", operator.ObjectMeta.Name, condition.Type, condition.Status, condition.Reason, condition.Message)
+				errorCount++
 			}
 		}
 	}
+	if(errorCount > 0){
+		logrus.Error(
+			"Cluster initialization failed because one or more operators are not functioning properly." +
+			" The cluster should be accessible for troublsehooting as detailed in the following documentation." +
+			" The 'wait-for install-complete' subcommand can then be used to continue the installation.\n\n" +
+			"https://docs.openshift.com/container-platform/latest/support/troubleshooting/troubleshooting-installations.html\n\n")
```

Comment:
I think we should just put this error [here](https://github.com/openshift/installer/blob/b894e4c734366194b4773ca4bbcf90a249c2a7e4/cmd/openshift-install/create.go#L408) instead. When install-complete fails due to timeout this error response is valid for almost all cases.

Reply:
secondly, adding `\n` at the end will force the output to look like

```
ERROR The 'wait-for install-complete' subcommand can then be used to continue the installation.

https://docs.openshift.com/container-platform/latest/support/troubleshooting/troubleshooting-installations.html

```

when it should look like
```
ERROR The 'wait-for install-complete' subcommand can then be used to continue the installation.
ERROR https://docs.openshift.com/container-platform/latest/support/troubleshooting/troubleshooting-installations.html
```

which keeps the UX consistent and also looks better when there are other INFO/ERROR lines around. it.
since logrus doesn't automatically do this https://github.com/sirupsen/logrus/issues/608 let's just call logrus.Error for each newline you want.